index.bs

```diff
@@ -3144,21 +3144,46 @@ partial interface MLGraphBuilder {
 </div>
 
 ### The prelu() method ### {#api-mlgraphbuilder-prelu}
-Calculate the <a href="https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Parametric_ReLU">parametric version of rectified linear function (Parametric Relu)</a> on the input tensor element-wise. Parametric Relu is a type of leaky ReLU that, instead of having a scalar slope like 0.01, making the slope (coefficient of leakage) into a parameter that is learned during the model training phase of this operation. The calculation follows the expression `max(0, x) + slope ∗ min(0, x)`.
+Calculate the <a href="https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Parametric_ReLU">parametric version of rectified linear function (Parametric ReLU)</a> on the input tensor element-wise. Parametric ReLU is a type of leaky ReLU that, instead of having a scalar slope like 0.01, making the slope (coefficient of leakage) into a parameter that is learned during the model training phase of this operation. The calculation follows the expression `max(0, x) + slope ∗ min(0, x)`.
 <script type=idl>
 partial interface MLGraphBuilder {
-  MLOperand prelu(MLOperand x, MLOperand slope);
+  MLOperand prelu(MLOperand input, MLOperand slope);
 };
 </script>
-<div algorithm=prelu>
+
+<div>
     **Arguments:**
-        - *x*: an {{MLOperand}}. The input tensor.
-        - *slope*: an {{MLOperand}}. The slope tensor. Its shape is either the same as, or unidirectionally broadcastable to the shape of input tensor *x* according to [[!numpy-broadcasting-rule]].
+        - *input*: an {{MLOperand}}. The input tensor.
+        - *slope*: an {{MLOperand}}. The slope tensor. Its shape is either the same as, or unidirectionally broadcastable to the shape of input tensor *input* according to [[!numpy-broadcasting-rule]].
 
     **Returns:**
         - an {{MLOperand}}. The output tensor of the same shape as *x*.
+</div>
 
-    <div class="note">
+<details open>
+  <summmary>
+    The {{MLGraphBuilder/prelu(input, slope)}} steps are:
+  </summary>
+  <div algorithm=prelu-input-slope class=algorithm-steps>
+    1. If |input| or |slope| is not an instance of {{MLOperand}}, then throw a "{{TypeError}}" {{DOMException}} and stop.
+    1. Let |descriptor| be a new {{MLOperandDescriptor}}.
+    1. Set |descriptor|.{{MLOperandDescriptor/dimensions}}.{{MLOperandDescriptor/type}} to |input|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/type}}.
+    1. Let |descriptor|.{{MLOperandDescriptor/dimensions}} be the result of running the [=MLGraphBuilder/broadcast-shapes=] steps given |input|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}} and |slope|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}}.
```

Comment:
The CI build fails with:

 `LINE ~3171: No 'dfn' refs found for 'broadcast-shapes'.`